lib/simp/beaker_helpers.rb

```diff
@@ -68,7 +68,16 @@ def copy_to(sut, src, dest, opts={})
 
       # End rsync hackery
 
-      rsync_to(sut, src, dest, _opts)
+      begin
+        rsync_to(sut, src, dest, _opts)
+      rescue
```

Comment:
Is there a specific exception that can be caught here?

Reply:
@lnemsick-simp No idea. This only captures `StandardError` so it's what I would do anyway. Basically if `rsync` fails *for any reason*, fall back.